templates/ruby/lib/http_client/error_handler.rb

```diff
@@ -27,7 +27,7 @@ def call(env)
             end
 {{if .Api.Response.Formats.Json}}
             # If JSON, a particular field is taken and used
-            if type.include?("json") and body.is_a?(Hash)
+            if type && type.include?("json") && body.is_a?(Hash)
```

Comment:
Yes, I understood that. My point is none of the current conditionals are going to be a problem is we use `and`. I do want to go through all of them later on change them to `&&`.